hieradata/node/perfsonar1.ls.lsst.org.yaml

```diff
@@ -118,6 +118,8 @@ network::mroutes_hash:
       "198.32.252.232/31": *via
       "198.32.252.234/31": *via
       "199.36.153.8/30": *via
+      "134.79.235.242/32": *via #slac routes
+      "134.79.235.226/32": *via #slac routes
```

Comment:
These routes presumably need to be different interfaces as iperf3 and owamp traffic are intended to be on different physical interfaces.

Reply:
See https://jira.lsstcorp.org/browse/IT-3892

Reply:
Per discussion on slack, it sounds like she should roll this out as-is.